www/js/controller.js

```diff
@@ -865,7 +865,7 @@ define([
 				console.log("Parser error: ", err);
 				console.log(err.message);
 				console.log(err.Error);
-				this.setAuthorStatus(this.currentID,'equation','incorrect');
+				this._model.authored.setAuthorStatus(this.currentID,'equation','incorrect');
```

Comment:
I am good either way. Let's just send this build. I will remember this because when we make student handlers this would be part of the flow. Put same code in controller.js